### ISSUE TITLE
go get -u cloud.google.com/go/iam/credentials/apiv1 failing

### ISSUE THREAD

**Issue by tsutsu** [googleapis/google-cloud-go#1504]
## Describe Your Environment

Go v1.12.7 on macOS, in a repo using Go Modules

## Expected Behavior

Library builds.

## Actual Behavior

```
$ go get -u cloud.google.com/go/iam/credentials/apiv1
go: finding cloud.google.com/go/iam/credentials/apiv1 latest
go: finding cloud.google.com/go/iam latest
[...other deps...]
# cloud.google.com/go/iam/credentials/apiv1
../../Library/Caches/org.golang/gopath/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:206:93: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenRequest
../../Library/Caches/org.golang/gopath/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:206:176: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenResponse
../../Library/Caches/org.golang/gopath/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:210:12: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenResponse
../../Library/Caches/org.golang/gopath/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:213:37: c.iamCredentialsClient.GenerateIdentityBindingAccessToken undefined (type "google.golang.org/genproto/googleapis/iam/credentials/v1".IAMCredentialsClient has no field or method GenerateIdentityBindingAccessToken)
```

## Cause

[This commit](https://github.com/googleapis/go-genproto/commit/3bdd9d9f5532d75d09efb230bd767d265245cfe5) of `go-genproto` removed the referenced APIs.

**Comment by jadekler** [comment]
I'm able to repro:

```
deklerk at deklerk-macbookpro2 in ~/workspace/test-iam
$ go mod init testiam
go: creating new go.mod: module testiam
deklerk at deklerk-macbookpro2 in ~/workspace/test-iam
$ go get -u cloud.google.com/go/iam/credentials/apiv1
go: finding cloud.google.com/go/iam/credentials/apiv1 latest
go: finding cloud.google.com/go/iam latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/sys latest
go: finding github.com/jstemmer/go-junit-report latest
go: finding golang.org/x/lint latest
go: finding github.com/google/pprof latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/net latest
go: finding golang.org/x/time latest
go: finding google.golang.org/genproto latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/crypto latest
go: finding golang.org/x/exp latest
go: finding honnef.co/go/tools v0.0.0-2019.2.1
go: finding golang.org/x/image latest
go: finding golang.org/x/mobile latest
go: finding github.com/golang/glog latest
go: finding github.com/BurntSushi/xgb latest
go: finding golang.org/x/exp v0.0.0-20190306152737-a1d7652674e8
go: downloading golang.org/x/net v0.0.0-20190628185345-da137c7871d7
go: extracting golang.org/x/net v0.0.0-20190628185345-da137c7871d7
# cloud.google.com/go/iam/credentials/apiv1
../go/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:206:93: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenRequest
../go/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:206:176: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenResponse
../go/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:210:12: undefined: "google.golang.org/genproto/googleapis/iam/credentials/v1".GenerateIdentityBindingAccessTokenResponse
../go/pkg/mod/cloud.google.com/go@v0.41.0/iam/credentials/apiv1/iam_credentials_client.go:213:37: c.iamCredentialsClient.GenerateIdentityBindingAccessToken undefined (type "google.golang.org/genproto/googleapis/iam/credentials/v1".IAMCredentialsClient has no field or method GenerateIdentityBindingAccessToken)
```

Working on a fix.

**Comment by jadekler** [comment]
@tsutsu Could you try adding `google.golang.org/genproto v0.0.0-20190708153700-3bdd9d9f5532` to your `go.mod` and seeing that fixes the problem?

**Comment by jadekler** [comment]
For posterity: This requires releasing a new version, AFAICT. Will kick that off on Monday.

**Comment by tsutsu** [comment]
@jadekler it does not (and, in fact, that was already what was in there.)

**Comment by jadekler** [comment]
Odd..

**Comment by jadekler** [comment]
Fixed in v0.42.0:

```
deklerk at deklerk in /tmp/foo
$ go mod init foo
go: creating new go.mod: module foo
deklerk at deklerk in /tmp/foo
$ go get -u cloud.google.com/go
go: finding cloud.google.com/go v0.42.0
go: finding golang.org/x/time latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/net latest
go: finding google.golang.org/genproto latest
go: finding golang.org/x/sys latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/exp latest
go: finding google.golang.org/grpc v1.22.0
go: finding golang.org/x/lint latest
go: finding github.com/BurntSushi/xgb latest
go: finding github.com/jstemmer/go-junit-report latest
go: finding github.com/golang/glog latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/image latest
go: finding honnef.co/go/tools v0.0.0-2019.2.1
go: finding golang.org/x/crypto latest
go: finding golang.org/x/mobile latest
go: finding github.com/google/pprof latest
go: finding honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc
go: finding golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135
go: finding golang.org/x/exp v0.0.0-20190306152737-a1d7652674e8
go: downloading cloud.google.com/go v0.42.0
go: extracting cloud.google.com/go v0.42.0
```
